src/shared/ui/logo/logo.tsx

```diff
@@ -0,0 +1,32 @@
+import { HTMLAttributes } from 'react';
+import { VariantProps, cva } from 'class-variance-authority';
+import classNames from 'classnames/bind';
+import logo from '@/shared/assets/svg/rss-logo.svg';
+import Image from '@/shared/ui/image';
+
+import styles from './logo.module.scss';
+
+type LogoProps = HTMLAttributes<HTMLElement> & VariantProps<typeof logoVariants>;
+
+export const cx = classNames.bind(styles);
+
+const logoVariants = cva(cx('logo'), {
+  variants: {
+    type: {
+      default: cx(''),
```

Comment:
I don't think we need `default: cx(''),` at all

Reply:
How could we use `defaultVariants` WO `type: default` in this case? E.g. in my component I have the same code:
![image](https://github.com/user-attachments/assets/b8ade4fd-7aa0-4eaa-b0b8-f93f40eeeca8)

Reply:
> How could we use `defaultVariants` WO `type: default` in this case? E.g. in my component I have the same code: ![image](https://private-user-images.githubusercontent.com/56927722/356872944-b8ade4fd-7aa0-4eaa-b0b8-f93f40eeeca8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjMzODIyNzYsIm5iZiI6MTcyMzM4MTk3NiwicGF0aCI6Ii81NjkyNzcyMi8zNTY4NzI5NDQtYjhhZGU0ZmQtN2FhMC00ZWFhLWIwYjgtZjkzZjQwZWVlY2E4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODExVDEzMTI1NlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ0ZjUwYjdlZTY2ZTdiOTBiMjA3NWI1MGFhYWIwNGFkMjNkZWFjZmNjM2VjNGI1NDVmOTJlNTU3NjAxYWJiZmEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.IzaDhd9Ee_SENtbeS_AF4anibh8Ajy-QV5rsD9DOTxU)

In this case, we can just not specify `type`

Reply:
later I use different types to check different test ids, I don't quite understand how to fix it

Reply:
fixed